CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 ## Unreleased
 
+- Changed dependency: Use `stylelint-prettier` for prettier integration instead of `prettier-stylelint-formatter`. `stylelint-prettier` is stylelint plugin that exposes prettier issues as stylelint rule violations. This means you can use `stylelint --fix` to fix formatting issues that prettier raises instead of having to use different executables for showing and autofixing issues.
```

Comment:
Nitpick: `stylelint-prettier` is **a** stylelint plugin that exposes...

Reply:
One day I will give the impression of being literate. Today is not that day.